### ISSUE TITLE
Keypair is not generated when user is created from son-gui

### ISSUE THREAD

**Issue by felipevicens** [sonata-nfv/son-gkeeper#964]
I've noticed that if the user is created from son-gui, (developer role) then this doesn't have a keypair.

So if the developer instantiate a NS, then it can't access using the default ssh keys injection.

**Comment by jbonnet** [comment]
Hi,
This is a feature, not a bug :-)
I thought we had agreed that only customers would have this key pair generated...

Is it not?
José

**Comment by felipevicens** [comment]
Not sure because demo user is a developer and has keypair created. Sonata user doesn't have keypair (Should have it). If we say that the developer will not have a keypair, then we should deny the NS instantiation for developers. right?

**Comment by jbonnet** [comment]
Yes, that was the idea: intantiating a service should be possible only through he BSS…

José Bonnet
jose.bonnet@me.com

**Comment by dang03** [comment]
Hi,
Our `demo` user is both `developer` and `customer`, and its keypair is generated due to the latest role.
In the case of `sonata` user, it is an `admin` user, and this role currently does not generate any keypair.
Do you think that an admin user should be able to instantiate a service?
